pkg/apis/serving/v1/revision_defaults.go

```diff
@@ -93,7 +93,15 @@ func (rs *RevisionSpec) SetDefaults(ctx context.Context) {
 		if rs.PodSpec.Containers[idx].ReadinessProbe.SuccessThreshold == 0 {
 			rs.PodSpec.Containers[idx].ReadinessProbe.SuccessThreshold = 1
 		}
+		if rs.PodSpec.Containers[idx].ReadinessProbe.PeriodSeconds != 0 {
+			if rs.PodSpec.Containers[idx].ReadinessProbe.FailureThreshold == 0 {
+				rs.PodSpec.Containers[idx].ReadinessProbe.FailureThreshold = 3
+			}
 
+			if rs.PodSpec.Containers[idx].ReadinessProbe.TimeoutSeconds == 0 {
+				rs.PodSpec.Containers[idx].ReadinessProbe.TimeoutSeconds = 10
```

Comment:
I think [default](https://github.com/knative/serving/blob/master//vendor/k8s.io/api/core/v1/types.go#L2022) value for `TimeoutSeconds` is `1 second` ?

Reply:
Sorry. My bad. Read the comments below the field. Have updated the PR.